File: examples/matlab/add_node_names.m
% define a couple edges
edges = [...
    1,2;
    2,3;...
    ];
% Place the edges in a webweb struct called ww
ww.networks.network.edgeList = edges;

% Define some names in an array of cells containing strings
names = {'Huberg','Pierot','Slartibertfast'};

% Put the names in metadata with the special key "names"
ww.display.metadata.name.values = names;

% Call webweb
webweb(ww)

File: examples/python/add_node_names.py
from webweb import Web

web = Web(
    adjacency=[[0, 1], [1, 2]],
    display={
        'nodes' : {
            0 : {
                'name' : 'Huberg',
            },
            1 : {
                'name' : 'Pierot',
            },
            2 : {
                'name' : 'Slartibertfast',
            },
        },
    },
)

# show the visualization
web.show()
